cfpb/consumerfinance.gov
# Wagtail: Remove heading and related posts from sidebar breakout

https://github.com/cfpb/consumerfinance.gov/pull/7927 made me notice that there are six options that can be added to a sidebar breakout: slug, paragraph, breakout image, job listing list, heading, and related posts.

The sidebar breakout is used on two pages:
https://www.consumerfinance.gov/rules-policy/
https://www.consumerfinance.gov/about-us/careers/

Neither of these pages uses the heading and related posts options, so this PR removes those as options.

## Removals

- Remove heading and related posts as options in the sidebar breakout

## How to test this PR

1. Pull branch
2. Run `python cfgov/manage.py migrate`
3. Visit http://localhost:8000/rules-policy/ and http://localhost:8000/about-us/careers/ and see that the pages are not broken. If you edit either in wagtail, see that under the sidebar > breakout sidebar selection only has four options (instead of six) to add.
